packages/Tabs/TabsWrapper.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { useRef, useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import FlexGrid from '@tds/core-flex-grid'
+import { ChevronRight, ChevronLeft } from '@tds/core-interactive-icon'
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs'
+import { TabsContainer, TabListContainer, TabLabel, TabArrows } from './styles'
+import hash from './hash'
+// import TabsElement from './TabsElement'
+/**
+ * @version ./package.json
+ * @visibleName Tabs (beta)
+ */
+
+const TabsWrapper = ({ tabsEntries, tabIndex }) => {
+  const tabsRoot = useRef()
+  const [tabsContainerWidth, setTabsContainerWidth] = useState()
+  const [tabsTranslatePosition, setTabsTranslatePosition] = useState(0)
+  const [isScrollEnabled, setScrollEnabled] = useState(false)
+  const [isLeftArrow, setIsLeftArrow] = useState(false)
+  const [isRightArrow, setIsRightArrow] = useState(false)
+  const fullWithRef = useRef(0)
+
+  const getTabsWidth = () => {
+    const tabsArray = document.querySelectorAll('.react-tabs__tab')
```

Comment:
Can we change this to a ref instead of using document.querySelectorAll

Reply:
I have made some changes in Tabs.jsx and removed document.querySelectorAll and using ref.